.circleci/config.yml

```diff
@@ -25,14 +56,24 @@ commands:
             pip install --progress-bar=off
               -e .
               -r tests_requirements.txt
-              tensorflow==2.6.* tensorflow-probability==0.14.* \n
+              tensorflow==<<parameters.tf_ver>>.*
```

Comment:
Can you omit the trailing `.*`? (in case you ever want a min/max version that includes the point version)

Reply:
> Can you omit the trailing `.*`? (in case you ever want a min/max version that includes the point version)

Hmm, it appears that I can.